### PR TITLE
Added home easy advanced

### PR DESCRIPTION
**Pull request by Xento**
When sending the message is one bit longer than the received one, but it
works.
Original: 110001111001011110001111001101001101110101010110101100011
Send: 1100011110010111100011110011010011011101010110110101100011

Could not test the groupfeature because I only have simple wallswitches
without "all on/off" button.
The unit seems to be the same on all switches.

### CODE REVIEW

File: lib/protocols/switch10.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	  groupcode = helper.binaryToNumber(binary, 43, 46);
+	  groupcode2 = helper.binaryToNumber(binary, 49, 50);
+	  
+	  if(groupcode == 11 && groupcode2 == 1) {
+		groupRes = false;
+	  } else if(groupcode == 3 && groupcode2 == 3) {
+		groupRes = true;
+	  }
+	  
+	  id2 = helper.binaryToNumber(binary, 11, 42)>>>0
```
Comment (sweetpi) [inline lib/protocols/switch10.js]: why this shift?

**Comment by sweetpi** [comment]
Could you convert this to a corresponding coffeescript file in `src`?

File: lib/protocols/switch10.js (hunk 1 above)
Comment (Xento) [inline lib/protocols/switch10.js]: Without this shift you get a negativ number.

**Comment by Xento** [comment]
I will try my best. How to compile it?
Is it the same as the other files that it is compiled when I start pimatic?

**Comment by sweetpi** [comment]
The compilation process can be find in the readme: https://github.com/pimatic/rfcontroljs#preparation

**Comment by Xento** [comment]
I now created the coffescript file

**Pull request merged**